docs/source/aws.rst

```diff
@@ -114,13 +114,13 @@ AWS VPC networks can be provisioned using this resource.
 >`_
 
 
-aws_ec2_vpc_net
```

Comment:
Are we removing vpc net support?

Reply:
no just fixing a bug on the way: duplicate document entries of aws_ec2_vpc_net (see lines 107-114)